pronotepy/test_pronotepy.py

```diff
@@ -84,6 +84,16 @@ def test_refresh(self) -> None:
         client.refresh()
         self.assertEqual(client.session_check(), True)
 
+    def test_report(self) -> None:
+        report = client.report(client.periods[0])
+
+        self.assertIsInstance(report, pronotepy.Report)
+
+    def test_not_published_report(self) -> None:
+        report = client.report(client.periods[3])
```

Comment:
Is this something we can count on being on the demonstration site? I'd say just make simple for loop, checking if it is `pronotepy.Report`, but let's try it and fix it later on if it stops working...